package.json

```diff
@@ -34,33 +34,33 @@
-		"docker:build": "docker run -it --rm -v \"${PWD}:/usr/src/app\" -w /usr/src/app \"node:$(cat .nvmrc)\" bash -c 'pnpm install && jetpack build packages && jetpack build plugins/jetpack'",
-		"docker:build-image": "docker build -t automattic/jetpack-wordpress-dev tools/docker",
-		"docker:clean": "pnpm run docker:compose -- down --rmi all -v && rm -rf tools/docker/wordpress/* tools/docker/wordpress/.htaccess tools/docker/wordpress-develop/* tools/docker/logs/* tools/docker/data/mysql/*",
-		"docker:compose": "pnpm run docker:env && pnpm run docker:compose-volumes && pnpm run docker:compose-extras && docker-compose -f tools/docker/docker-compose.yml -f tools/docker/compose-volumes.built.yml -f tools/docker/compose-extras.yml",
-		"docker:compose-extras": "[ -f tools/docker/compose-extras.yml ] || cp tools/docker/compose-extras.yml.sample tools/docker/compose-extras.yml",
-		"docker:compose-volumes": "[ -f tools/docker/compose-volumes.yml ] || cp tools/docker/compose-volumes.yml.sample tools/docker/compose-volumes.yml; bash ./tools/compose-volumes.sh tools/docker/compose-volumes.yml > tools/docker/compose-volumes.built.yml",
-		"docker:db": "pnpm run docker:compose -- exec wordpress bash -c \"mysql --defaults-group-suffix=docker\"",
-		"docker:down": "pnpm run docker:compose -- down",
-		"docker:env": "node -e \"require('fs').createWriteStream( 'tools/docker/.env', { flags: 'a' } );\"",
-		"docker:install": "pnpm run docker:compose -- exec wordpress bash -c \"/var/scripts/install.sh\"",
-		"docker:jt-config": "[ -f ./tools/docker/bin/jt/config.sh ] || { echo 'Tunneling scripts are not installed. See the section \"Jurassic Tube Tunneling Service\" in tools/docker/README.md.'; exit; }; ./tools/docker/bin/jt/config.sh",
-		"docker:jt-down": "[ -f ./tools/docker/bin/jt/tunnel.sh ] || { echo 'Tunneling scripts are not installed. See the section \"Jurassic Tube Tunneling Service\" in tools/docker/README.md.'; exit; }; ./tools/docker/bin/jt/tunnel.sh break",
-		"docker:jt-up": "[ -f ./tools/docker/bin/jt/tunnel.sh ] || { echo 'Tunneling scripts are not installed. See the section \"Jurassic Tube Tunneling Service\" in tools/docker/README.md.'; exit; }; ./tools/docker/bin/jt/tunnel.sh",
-		"docker:multisite-convert": "pnpm run docker:compose -- exec wordpress bash -c \"/var/scripts/multisite-convert.sh\"",
+		"docker:build": "printf '\\e[30;43m`pnpm run docker:build` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker build",
+		"docker:build-image": "printf '\\e[30;43m`pnpm run docker:build-image` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker build-image",
+		"docker:clean": "printf '\\e[30;43m`pnpm run docker:clean` is deprected. Use the Jetpack CLI instead.\\e[0m\\n'; jetpack docker clean",
+		"docker:compose": "printf '\\e[30;43m`pnpm run docker:compose` is deprected. Now it is integrated into relevant Jetpack CLI commands.\\e[0m\\n';",
```

Comment:
The JT scripts currently call `pnpm run docker:compose -- exec wordpress ...` to try to run a command inside the container. So now I get
```
$ jetpack docker jt-up
Establishing the connection

> Jetpack_Monorepo@ docker:compose /usr/local/src/automattic/jetpack
> printf '\e[30;43m`pnpm run docker:compose` is deprected. Now it is integrated into relevant Jetpack CLI commands.\e[0m\n'; "exec" "wordpress" "/var/scripts/jt/jtuber.sh" "init" "8702" "localhost:80" "bjorsch"

`pnpm run docker:compose` is deprected. Now it is integrated into relevant Jetpack CLI commands.
sh: 1: exec: wordpress: not found
 ERROR  Command failed.
```

Reply:
I opened a PR in JN to switch to `jetpack cli` command: 3-gh-Automattic/jurassic-tube